<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>1200</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::CustomContextMenu</enum>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextUnderIcon</enum>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionFileOpen"/>
    <addaction name="actionFileOpenRecent"/>
    <addaction name="separator"/>
    <addaction name="actionFileSave"/>
    <addaction name="actionFileSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionFilePrint"/>
    <addaction name="separator"/>
    <addaction name="actionFileClose"/>
    <addaction name="actionFileCloseAll"/>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionViewFullScreen"/>
   </widget>
   <widget class="QMenu" name="menuOFD">
    <property name="title">
     <string>OFD</string>
    </property>
    <widget class="QMenu" name="menuDisplay">
     <property name="title">
      <string>Display</string>
     </property>
     <addaction name="actionOFDSinglePage"/>
     <addaction name="actionOFDSinglePageContinuous"/>
     <addaction name="actionOFDTwoPages"/>
     <addaction name="actionOFDTwoPagesContinuous"/>
    </widget>
    <addaction name="menuDisplay"/>
    <addaction name="separator"/>
    <addaction name="actionOFDAutomaticallyResize"/>
    <addaction name="separator"/>
    <addaction name="actionOFDActualSize"/>
    <addaction name="actionOFDZoomToFit"/>
    <addaction name="actionOFDZoomIn"/>
    <addaction name="actionOFDZoomOut"/>
    <addaction name="actionOFDZoomToSelection"/>
    <addaction name="separator"/>
    <addaction name="actionOFDCurrentSettingsDefault"/>
   </widget>
   <widget class="QMenu" name="menu_Tools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionToolsTextTool"/>
    <addaction name="actionToolsLineTool"/>
   </widget>
   <widget class="QMenu" name="menuBookmarks">
    <property name="title">
     <string>Bookmarks</string>
    </property>
    <addaction name="actionBookmarksAddBookmark"/>
    <addaction name="separator"/>
    <addaction name="actionBookmarksManage"/>
   </widget>
   <widget class="QMenu" name="menuWindows">
    <property name="title">
     <string>Window</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelpQTOFDReaderHelp"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_View"/>
   <addaction name="menuOFD"/>
   <addaction name="menu_Tools"/>
   <addaction name="menuBookmarks"/>
   <addaction name="menuWindows"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionFileOpen">
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionFileOpenRecent">
   <property name="text">
    <string>Open Recent</string>
   </property>
  </action>
  <action name="actionFileClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionFileCloseAll">
   <property name="text">
    <string>Close All</string>
   </property>
  </action>
  <action name="actionFileSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionFileSaveAs">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actionFilePrint">
   <property name="text">
    <string>Print...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionViewFullScreen">
   <property name="text">
    <string>Full Screen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+F</string>
   </property>
  </action>
  <action name="actionOFDSinglePage">
   <property name="text">
    <string>Single Page</string>
   </property>
  </action>
  <action name="actionOFDSinglePageContinuous">
   <property name="text">
    <string>Single Page Continous</string>
   </property>
  </action>
  <action name="actionOFDTwoPages">
   <property name="text">
    <string>Two Pages</string>
   </property>
  </action>
  <action name="actionOFDTwoPagesContinuous">
   <property name="text">
    <string>Two Pages Continous</string>
   </property>
  </action>
  <action name="actionOFDAutomaticallyResize">
   <property name="text">
    <string>Automatically Resize</string>
   </property>
  </action>
  <action name="actionOFDActualSize">
   <property name="text">
    <string>Actual Size</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="actionOFDZoomToFit">
   <property name="text">
    <string>Zoom To Fit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
  <action name="actionOFDZoomIn">
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+=</string>
   </property>
  </action>
  <action name="actionOFDZoomOut">
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionOFDZoomToSelection">
   <property name="text">
    <string>Zoom To Selection</string>
   </property>
  </action>
  <action name="actionOFDCurrentSettingsDefault">
   <property name="text">
    <string>Use Current View Settings as Default</string>
   </property>
  </action>
  <action name="actionToolsTextTool">
   <property name="text">
    <string>Text Tool</string>
   </property>
  </action>
  <action name="actionToolsLineTool">
   <property name="text">
    <string>Line Tool</string>
   </property>
  </action>
  <action name="actionBookmarksAddBookmark">
   <property name="text">
    <string>Add Bookmark</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionBookmarksManage">
   <property name="text">
    <string>Edit Bookmarks...</string>
   </property>
  </action>
  <action name="actionHelpQTOFDReaderHelp">
   <property name="text">
    <string>qtOFDReader Help</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionViewFullScreen</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showFullScreen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>599</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
